src/components/wizards/new-vm/NewVmWizard.stories.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { NewVmWizard } from './NewVmWizard';
+import { Button } from 'patternfly-react';
+import { os } from './OperatingSystem';
+
+class NewVmWizardExample extends React.Component {
+  state = {
+    show: false
+  }
+
+  storageClasses = [
```

Comment:
maybe move to a fixture files? could be reused in tests?

Reply:
sounds good, TODO